### ISSUE TITLE
GeoJSON or TopoJSON tool

### ISSUE THREAD

**Issue by freewheelCarto** [rhaugerud/GeMS_Tools#11]
Has anyone considered an export to GeoJSON or TopoJSON? I am thinking of looking into it, but wanted to make sure that it wasn't already considered and decided against. 

I think it is potentially a good open alternative format. I know ArcGIS has export options to ESRI JSON standard, and with 10.5 you can use their tool to export to GeoJSON spec (I still have 10.3). However, I don't know if just running through the features and tables with that tool would produce an ideal export.

**Comment by ethoms-usgs** [comment]
Yes! I think it should be considered. I would like to see a plain-text format being used, if only for archiving. I have played around a bit with both although I didn't take a close look at how the attributes got exported. GeoJSON is pretty cool in QGIS. You can add it through drag and drop and save edits back out to the original text file. TopoJSON is very intriguing but, at least when I looked about 8 months ago, QGIS didn't edit the format properly. It will display the two layers that are in the topology but when editing only changes to the linework were saved, not the polygons. I used mapshaper.org to build TopoJSON files. 

So, it might be a bit early to push for TopoJSON, but I think we should look at GeoJSON and provide a tool and/or guidance.

**Comment by freewheelCarto** [comment]
When I get a little more time I'll look at some export options for GeoJSON. A couple of questions about it that I have are:

- Could you save the stand alone tables as JSON files and do joins or relates to the GeoJSON (in QGIS)? 
- Will the export need to be programmed with custom scripts, or can the existing ArcGIS Conversion tools do what's needed?
- If the existing tools can do it, does there need to be any other work to prepare it for a proper GeoJSON export?
